app/page/template/video-calling.htm

```diff
@@ -51,8 +51,16 @@
               data-bind="click: clickedOnStopVideo, css: {'video-controls__button--active': selfStreamState.videoSend()}, attr: {'data-uie-value': selfStreamState.videoSend() ? 'active' : 'inactive'}"
               data-uie-name="do-call-controls-toggle-video">
                 <camera-icon></camera-icon>
-                <div class="video-controls__button__label" data-bind="l10n_text: z.string.videoCallOverlayVideo"></div>
-
+                <!-- ko if: showSwitchCamera() -->
+                  <device-toggle-button data-bind="click: clickedOnToggleCamera"
+                    params="index: currentDeviceIndex.video_input,
+                      devices: availableDevices.video_input,
+                      type: z.media.MediaDeviceType.VIDEO_INPUT">
+                  </device-toggle-button>
+                <!-- /ko -->
+                <!-- ko ifnot: showSwitchCamera -->
```

Comment:
Maybe `showSwitchCamera()` would be better, so that we don't, when reading the code, testing that the observable exists with testing the actual value of the observable.